### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#147] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-09 172745](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4dc95a1e-16bc-45a7-99d0-0addc6807dc5)

**Comment by daromacarlo** [comment, marked as answer]
![Immagine 2023-12-16 163342](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/f362eea6-6229-4780-b6ba-f3286b657ca5)
su questo secondo metodo ho molti dubbi.
![Immagine 2023-12-16 163353](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/5e9c5142-a1da-48b9-8fd1-45ff856a4176)
